src/tokenManagement/tokenManagementTransactionsFactory.ts

```diff
@@ -705,19 +590,15 @@ export class TokenManagementTransactionsFactory {
 
         return new TransactionBuilder({
             config: this.config,
-            sender: options.sender,
-            receiver: options.sender,
+            sender: sender,
+            receiver: sender,
             dataParts: dataParts,
             gasLimit: this.config.gasLimitSetNewUris,
             addDataMovementGas: true,
         }).build();
     }
 
-    createTransactionForModifyingCreator(options: {
-        sender: IAddress;
-        tokenIdentifier: string;
-        tokenNonce: bigint;
-    }): Transaction {
+    createTransactionForModifyingCreator(sender: IAddress, options: resources.BaseInput): Transaction {
```

Comment:
Not very explicit naming. Could've extended  `BaseInput` to have another name.

Reply:
Updated